<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>517</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>517</width>
    <height>458</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>441</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_acceuil">
     <attribute name="title">
      <string>ACCUEIL</string>
     </attribute>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ACCUEIL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_envoyer">
     <attribute name="title">
      <string>ENVOYER</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_contacts">
     <attribute name="title">
      <string>CONTACTS</string>
     </attribute>
     <widget class="QLineEdit" name="contacts_input_entreprise">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contacts_input_adresse">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contacts_input_telephone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contacts_input_mail">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="contacts_label_entreprise">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ENTREPRISE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="contacts_label_contact">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTACT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="contacts_label_telephone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TELEPHONE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="contacts_label_adresse">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ADRESSE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="contacts_label_mail">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="contacts_textEdit_adresse">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="contacts_btn_save">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>372</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SAUVEGARDER</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>ENTREPRISE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ADRESSE</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="contacts_label_adresse_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>COMMENTAIRE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="contacts_textEdit_adresse_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>491</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="contacts_combobox_sexe">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Mme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_option">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_option"/>
  </widget>
  <action name="actionLocalisation_Images">
   <property name="text">
    <string>Localisation Images</string>
   </property>
  </action>
  <action name="actionConnexion_MYSQL">
   <property name="text">
    <string>Connexion MYSQL</string>
   </property>
  </action>
  <action name="actionConnexion_ACCESS">
   <property name="text">
    <string>Connexion ACCESS</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>?</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
